test/service-object.ts

```diff
@@ -817,24 +799,15 @@ describe('ServiceObject', () => {
     });
   });
 
-  describe('request', () => {
-    let request_;
-
-    before(() => {
-      request_ = ServiceObject.prototype.request_;
-    });
-
-    after(() => {
-      ServiceObject.prototype.request_ = request_;
-    });
+  describe.skip('request', () => {
```

Comment:
The function it's testing is:

```js
ServiceObject.prototype.request = function(reqOpts, callback) {
  ServiceObject.prototype.request_.call(this, reqOpts, callback);
};
```

The test is seeing that calling `request()` actually is calling `request_`. If it stopped working, maybe the `sandbox.stub` method is too late, or somehow not overriding the same way.

As far as why it's important to call `ServiceObject.prototype.request_.call(this, reqOpts, callback)` and not just `this.request_()`... I'm not confident enough to say for sure. As a guess, it might be to prevent an extending class from overriding `request_`, eg. `Bigtable.prototype.request_`, in which case the original `Service.prototype.request_` would not have been called.